compiler/native/compile.go

```diff
@@ -276,12 +291,6 @@ func (c *client) compileInline(p *yaml.Build, localTemplates []string) (*yaml.Bu
 		}
 	}
 
-	// validate the yaml configuration
-	err := c.Validate(&newPipeline)
-	if err != nil {
-		return nil, err
-	}
-
```

Comment:
was this intentional?

Reply:
Yes, I saw that both calls to `compileInLine` validate the result: https://github.com/go-vela/server/blob/5a9a92d279e058b75b37e589e1eef2ec63f550a6/compiler/native/compile.go#L89-L93 and https://github.com/go-vela/server/blob/5a9a92d279e058b75b37e589e1eef2ec63f550a6/compiler/native/compile.go#L124-L128. And I want to validate whatever is finally returned by that function rather than fragments as the function rolls back up.